src/pages/Statistics/StatisticsCities.tsx

```diff
@@ -421,33 +421,50 @@ const onClickReset = () => {
       </div>
       <br/>
       {!showTable ? "" :
-        <Table
-          bordered 
-          rowClassName={(record, index) => index === onClickRow ? "onClickRow" : "" }
-          rowKey="id"
-          columns={columns}
-          dataSource={dataForTable}
-          scroll={{ x: 1000 }}
-          onRow={(cityRecord, index) => {
-            return {              
-              onClick: async () => {      
-                setShowDataChart(true);        
-                setDataFromRow(cityRecord);
-                setOnClickRow(index);
-              },
-              onDoubleClick: async () => {                
-                setOnClickRow(null);
-              }
-            };
-          }}
-          pagination={{
-            showLessItems: true,
-            responsive: true,
-            showSizeChanger: true,
-          }}
-        />}        
-      </div>
-    </Layout.Content>
+      <Form>
+        <Form.Item>
+          <Row style={{fontSize: '20px'}} gutter={[5, 0]}>
```

Comment:
It is bad practise to use inline styles.
Please change it

Reply:
Thanks, changed